Dockerfile

```diff
@@ -20,5 +20,9 @@ RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
 
 COPY . /code/
 
-# Heroku uses PORT, Azure App Services uses WEBSITES_PORT, Fly.io uses 8080 by default
-CMD ["sh", "-c", "uvicorn server.main:app --host 0.0.0.0 --port ${PORT:-${WEBSITES_PORT:-8080}}"]
+ARG RENDER_EXTERNAL_HOSTNAME
+RUN grep -rl "your-app-url.com" . | xargs sed -i "s/your-app-url.com/${RENDER_EXTERNAL_HOSTNAME}/g"
+
+ARG WEAVIATE_HOSTNAME
+# Render and Heroku use PORT, Azure App Services uses WEBSITES_PORT, Fly.io uses 8080 by default
+CMD ["sh", "-c", "env WEAVIATE_HOST=http://${WEAVIATE_HOSTNAME} uvicorn server.main:app --host 0.0.0.0 --port ${PORT:-${WEBSITES_PORT:-8080}}"]
```

Comment:
This still works for other deployment options and vector stores, right?

Reply:
Other vector stores, yes, because we just introduced the `WEAVIATE_HOSTNAME` env var, which isn't used anywhere besides here. Other deployment options might break from us updating the app URL on line 25, but we'd make sure to generalize that before we attempt to upstream anything to the main repo maintained by OpenAI.